src/compo/viirs_l1bnc2ioda.py

```diff
@@ -185,12 +190,15 @@ def _read(self):
                 err.set_auto_scale(False)
 
                 # NASA VIIRS apply secant if option enabled
-                vals[:, ichan] = obs[:].data.ravel() * sec_term if self.apply_secterm else obs[:].data.ravel()
+                vals[:, ichan] = obs[:].data.ravel() * sec_term if not self.stored_refl else obs[:].data.ravel()
                 qcfs[:, ichan] = qcf[:].data.ravel()
-                errs[:, ichan] = 1. + err.scale_factor * err[:].data.ravel() ** 2
+                # Convert Uncertainty Index (in percentage) into absolute error
+                errs[:, ichan] = (1. + err.scale_factor * err[:].data.ravel() ** 2) * 0.01 * vals[:, ichan]
```

Comment:
Is this formula from the VIIRS L1b ATBD? Or is it something you came up with? Either specify where does it come from or justify it in a comment above it birefly.

Reply:
Following our meeting can the error be scaled to account for the secant term instead of multiplying O in the converter and then B in UFO?
Also, would an obs function be more appropriate for applying the secant term either on obs error or O-B ?
@BenjaminRuston is there such a thing in obs functions?

Reply:
I will remove secant term from obs value and apply it to obs error.